frontend/src/metabase/dashboard/actions/save.js

```diff
@@ -70,21 +86,16 @@ export const saveDashboardAndCards = createThunkAction(
               const result = await DashboardApi.addcard({
                 dashId: dashboard.id,
                 cardId: dc.card_id,
-              });
-              dispatch(updateDashcardId(dc.id, result.id));
-
-              // mark isAdded because addcard doesn't record the position
-              return {
-                ...result,
                 col: dc.col,
                 row: dc.row,
                 size_x: dc.size_x,
                 size_y: dc.size_y,
                 series: dc.series,
                 parameter_mappings: dc.parameter_mappings,
                 visualization_settings: dc.visualization_settings,
-                isAdded: true,
-              };
+              });
+              dispatch(updateDashcardId(dc.id, result.id));
+              return result;
```

Comment:
We just return result now since it contains all the previously required data (like col, row etc) now